docs/concepts/server-configuration.md

```diff
@@ -93,3 +99,30 @@ you can configure this to `https://www.example.com/about/` by using the
 ```ts
 await start(manifest, { router: { trailingSlash: true } });
 ```
+
+### basePath
+
+This setting allows you to serve a Fresh app from sub-path of a domain. A value
+of `/foo/bar` would serve the app from `http://localhost:8000/foo/bar` instead
+of `http://localhost:8000/` for example.
+
+This setting can also be set via the `FRESH_BASE_PATH` environment variable. It
```

Comment:
I don't think we need to add an env var for this OOTB. If someone really wants to do this with env var config, they can set `basePath: Deno.env.get("BASE_PATH")`.